test/test-style.js

```diff
@@ -2,6 +2,26 @@
 const fs = require('fs');
 const path = require('path');
 
+function orderSupportBlock(name, val) {
+    if (name == 'support') {
```

Comment:
**⚠️ WARNING! ⚠️** This will break any value named `support`, eg. `test.support`:
```json
"test": {
  "support": {
    "property": {},
    "method": {}
  }
}
```
would be made invalid, whereas:
```json
"test": {
  "other": {
    "property": {},
    "method": {}
  }
}
```
would remain valid.